### DISCUSSION TITLE
Welcome to fosstrak-india Discussions!

### DISCUSSION THREAD

**Discussion by zhravan** [shravan20/fosstrak-india#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by pk0dev** [comment]
I'm here to motivate you to build this project and share about your journey on LinkedIn and X.
